src/apps/loyalty/server/enroll_loyalty_applicant.ts

```diff
@@ -0,0 +1,18 @@
+import fetch from "node-fetch"
+
+export function fetchCollectorProfile(accessToken: string): any {
```

Comment:
right now TS is saying "well, I know nothing about fetch, so we'll let it do whatever it wants"